integration_tests/e2e/goal/review.cy.ts

```diff
@@ -94,6 +95,55 @@ context('Review goal(s)', () => {
     Page.verifyOnPage(AddNotePage)
   })
 
+  it('should be able to create another goal from the Review screen', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    const overviewPage = Page.verifyOnPage(OverviewPage)
+
+    let createGoalPage = overviewPage.clickAddGoalButton()
+    createGoalPage //
+      .setGoalTitle('Learn French')
+      .submitPage()
+
+    let addStepPage = Page.verifyOnPage(AddStepPage)
+    addStepPage //
+      .setStepTitle('Book French course')
+      .setStepTargetDateRange('ZERO_TO_THREE_MONTHS')
+      .submitPage()
+
+    let addNotePage = Page.verifyOnPage(AddNotePage)
+    addNotePage.setNote("Pay close attention to Chris' behaviour during classes")
+    addNotePage.submitPage()
+
+    const reviewPage = Page.verifyOnPage(ReviewPage)
+    cy.get('.govuk-summary-card').should('have.length', 1)
```

Comment:
This will do for now (we need to get this merged), but our general pattern is that the test itself knows very little about the underlying page driver technology, and that apart from the early `cy.task` type calls, we dont use `cy` in the test class itself.
The problem is that it couples the test too tightly to the page driver - what if we were to change from cypress to selenium or a.n.other web/page driver technology.
This is why we have the `Page` classes, that abstract the page driver implementation away from the test itself.
So really the `cy.get('.govuk-summary-card').should('have.length', 1)` should be in a suitably named method in `ReviewPage` - something like `hasNumberOfGoalsDisplayed(expected: number)`, which in turn calls the `cy` elements